src/main/java/com/google/growpod/servlets/UserServlet.java

```diff
@@ -134,4 +157,19 @@ private User getUserById(String id) {
     }
     return USER_GARDEN_LIST_MAP.get(id);
   }
+
+  /**
+   * Retrieves a list of gardens the user with a given id administers. Returns an empty list if the
+   * user administers no gardens, and null if the user does not exist.
+   *
+   * @param id the user's id
+   * @return a list of gardens the user administers, or an empty list, or null.
+   */
+  private List<String> getUserGardenAdminListById(String id) {
+    // MOCKUP IMPLEMENTATION
```

Comment:
nit: I don't think mockup is the longform of mock. i think they may be different words with a similar root. "Mock implementation" makes more sense imo.

Reply:
By "Mockup Implementation", I intended to communicate "Implementation for the Mockup", but "Mock Implementation"'s better.